internal/pkg/skuba/addons/cilium.go

```diff
@@ -87,235 +79,524 @@ func (ciliumCallbacks) afterApply(addonConfiguration AddonConfiguration, skubaCo
 
 const (
 	ciliumManifest = `---
+# Source: cilium/charts/config/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cilium-config
+  namespace: kube-system
+data:
+
+  # Identity allocation mode selects how identities are shared between cilium
+  # nodes by setting how they are stored. The options are "crd" or "kvstore".
+  # - "crd" stores identities in kubernetes as CRDs (custom resource definition).
+  #   These can be queried with:
+  #     kubectl get ciliumid
+  # - "kvstore" stores identities in a kvstore, etcd or consul, that is
+  #   configured below. Cilium versions before 1.6 supported only the kvstore
+  #   backend. Upgrades from these older cilium versions should continue using
+  #   the kvstore by commenting out the identity-allocation-mode below, or
+  #   setting it to "kvstore".
+  identity-allocation-mode: crd
+
+  # If you want to run cilium in debug mode change this value to true
+  debug: "false"
```

Comment:
Before it was possible to configure debug with CILIUM_DEBUG. Not sure if you remove that on purpose or you missed it. I don't think it is essential, we can always change this parameter to true and redeploy but still.

Reply:
CILIUM_DEBUG environment variable is feeded with the value from ConfigMap. So that value from that line finally goes to CILIUM_DEBUG.